mobile/src/Student/screens/Game/PhaseOneBasicGamePlay/index.js

```diff
@@ -299,17 +306,15 @@ const styles = StyleSheet.create({
         fontSize: fonts.small,
         fontWeight: fontWeights.extraBold,
         textAlign: "center",
-        marginHorizontal: scale(20),
-        marginVertical: scale(20),
     },
     hintsView: {
-        marginTop: -60,
+        marginTop: "-15%",
+        height: "110%"
```

Comment:
Looks like `height` is new, was this needed? I think previously the height was dynamic and expanded by the content, does this behave the same?